src/containers/app/AppEpic.js

```diff
@@ -6,18 +6,183 @@ import { AppApi } from 'lattice';
 import * as actionTypes from './AppActionTypes';
 import * as actionFactory from './AppActionFactory';
 
+
+function createAppTypeEpic(action$) {
+  return action$
+    .ofType(actionTypes.CREATE_APP_TYPE_REQUEST)
+    .mergeMap((action) => {
+      return Observable
+        .from(AppApi.createAppType(action.AppType))
```

Comment:
This doesn't currently exist in `lattice-js`. As such, this PR cannot be merged until the changes in https://github.com/openlattice/lattice-js/pull/413 are released. The same goes for most of the `AppApi` functions here.